source/s_graphs_color.ptx

```diff
@@ -88,7 +88,7 @@
           \bfG</m> is connected, as each component may be colored individually. The <term>distance</term> <m>
           d(u,v)</m> between vertices <m>u,v\in V</m> is the length of a shortest path from <m>u</m>
           to <m>v</m>, and of course <m>d(u,u) = 0</m>. Fix a vertex <m>v_0\in V</m> and define <me>A
-          = \{v\in V\colon d(u_0,v)\text{ is even}\}\qquad\text{and}\qquad B = \{v\in V\colon
+          = \{v\in V\colon d(v_0,v)\text{ is even}\}\qquad\text{and}\qquad B = \{v\in V\colon
```

Comment:
Given that distance is defined as d(u,v), I think it would be better to leave u_0 here and put u_0 in line 90 in place of v_0. If you push an update to update to that change, I would accept the PR.

Reply:
But later you have referred to $v_0, v_1$ etc in the text. If we use $u_0$, then later text also has to be altered. The easiest fix is to change $u_0$ to $v_0$ here.